src/main/java/mate/academy/model/CinemaHall.java

```diff
@@ -0,0 +1,57 @@
+package mate.academy.model;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "cinema_hall")
```

Comment:
```suggestion
@Table(name = "cinema_halls")
```
you use plural in other entities, you it here too